packages/driver/ts/import-coffee.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.coffee' {
```

Comment:
add comment why this file is needed: to make typescript ok with .coffee resolves